tor/archiver.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import logging
+
+import prawcore
+from praw import Reddit
+
+from tor import config
+from tor.core.initialize import configure_logging
+from tor.core.initialize import configure_tor
+from tor.helpers.misc import explode_gracefully, subreddit_from_url
```

Comment:
I've seen it done with parentheses like this in several codebases and it seems to follow your requirement. Does this version bother you?

```
from foo import (
   Fizz,
   Buzz,
   LoremIpsum,
)
```